src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num) {
+      calculator.result += num;
+    },
+    subtract(num) {
+      calculator.result -= num;
+    },
+    multiply(num) {
+      calculator.result *= num;
+    },
+    divide(num) {
+      calculator.result /= num;
+    },
```

Comment:
It is necessary to add a check by 0, because according to the rules, you cannot divide by 0